Add readme for inc 3.0 xpu device usage (#1693)
intel/intel-extension-for-transformers: examples/huggingface/pytorch/text-generation/quantization/README.md

## Run
The followings are command to show how to use it.

> > Note: if neural-compressor version is 3.0, please ```export FORCE_DEVICE="cpu"```, it is used to ensure calibration on cpu device.

Reviewer: ```suggestion
> Note: If the `neural-compressor` version is higher than 3.0, please ```export FORCE_DEVICE="cpu"``` to ensure that the quantization process runs on the cpu device.
```

Author: Don't you plan to improve this strange usage in INC in the future?
